susemanager/susemanager.changes

```diff
@@ -1,3 +1,6 @@
+- Reorganize bootstrap SSL state
+- Add missing packages on SSL bootstrap of Debian-10 and SLES-15
+- Update translation strings
```

Comment:
You are missing an empty line on top of the separator, and below this line.

But anyway, why this is needed? I don't remember the reorganization of the bootstrap states was backported to 4.2. Are you sure you are adding the right changes!?